src/Dfc.CourseDirectory.Web/Controllers/CoursesController.cs

```diff
@@ -49,141 +49,82 @@ public class CoursesController : Controller
-                   DurationUnit = DurationUnit.Months,
-                   DurationValue = 47,
-                   StudyMode = StudyMode.Flexible,
-                   AttendancePattern = AttendancePattern.DayOrBlockRelease,
-                   CreatedDate = Convert.ToDateTime("2019-01-03T11:17:02.514746+00:00"),
-                   CreatedBy = "ProviderPortal-AddCourse",
-                   UpdatedDate = Convert.ToDateTime("0001-01-01T00:00:00"),
-                   UpdatedBy = null
-               }
+                new CourseRun
+                {
+                      id = Guid.NewGuid(),
+                      VenueId = new Guid("76748623-93f0-4ebd-8a37-0f0754822b7e"),
+                      CourseName = "GCE A Level in Further Mathematics 1",
+                      ProviderCourseID = "asfdf-someId-courseId-string-guid",
+                      DeliveryMode = DeliveryMode.ClassroomBased,
+                      FlexibleStartDate = false,
+                      StartDate = Convert.ToDateTime("2021-04-03T00:00:00"),
+                      CourseURL = "http://www.bbc.co.uk",
+                      Cost = 125,
+                      CostDescription = "cost description",
+                      DurationUnit = DurationUnit.Months,
+                      DurationValue = 47,
+                      StudyMode = StudyMode.Flexible,
+                      AttendancePattern = AttendancePattern.DayOrBlockRelease,
+                      CreatedDate = Convert.ToDateTime("2019-01-03T11:17:02.514746+00:00"),
+                      CreatedBy = "ProviderPortal-AddCourse",
+                      UpdatedDate = Convert.ToDateTime("0001-01-01T00:00:00"),
+                      UpdatedBy = null
+                },
+                new CourseRun
+                {
+                    id = Guid.NewGuid(),
+                    VenueId = new Guid("36ea2887-31ac-48cf-9d99-d267c5d464e6"),
+                    CourseName = "GCE A Level in Further Mathematics 2",
+                    ProviderCourseID = "asfdf-someId-courseId-string-guid",
+                    DeliveryMode = DeliveryMode.WorkBased,
+                    FlexibleStartDate = true,
+                    //StartDate = Convert.ToDateTime("2021-04-03T00:00:00"),
+                    CourseURL = "http://www.bbc.co.uk",
+                    Cost = 125,
+                    CostDescription = "cost description",
+                    DurationUnit = DurationUnit.Weeks,
+                    DurationValue = 47,
+                    StudyMode = StudyMode.FullTime,
+                    AttendancePattern = AttendancePattern.Evening,
+                    CreatedDate = Convert.ToDateTime("2019-01-03T11:17:02.514746+00:00"),
+                    CreatedBy = "ProviderPortal-AddCourse",
+                    UpdatedDate = Convert.ToDateTime("0001-01-01T00:00:00"),
+                    UpdatedBy = null
+                }
             };
+           
+            
             Course[] course = new Course[]
             {
```

Comment:
Can you please copy this stub data over so the page looks more populated until we get the API sorted?